tests/conftest.py

```diff
@@ -136,8 +136,37 @@ def default_credentials(tmp_path: Path) -> Path:
     auth_file = tmp_path / 'default.toml'
     auth_file.write_text(
         """
-        [tokens."https://logfire-api.pydantic.dev"]
-        token = "0kYhc414Ys2FNDRdt5vFB05xFx5NjVcbcBMy4Kp6PH0W"
+        [tokens."https://logfire-us.pydantic.dev"]
+        token = "pylf_v1_us_0kYhc414Ys2FNDRdt5vFB05xFx5NjVcbcBMy4Kp6PH0W"
+        expiration = "2099-12-31T23:59:59"
+        """
+    )
+    return auth_file
+
+
+@pytest.fixture
+def expired_credentials(tmp_path: Path) -> Path:
+    auth_file = tmp_path / 'default.toml'
+    auth_file.write_text(
+        """
+        [tokens."https://logfire-us.pydantic.dev"]
+        token = "pylf_v1_us_0kYhc414Ys2FNDRdt5vFB05xFx5NjVcbcBMy4Kp6PH0W"
+        expiration = "1970-01-01T00:00:00"
+        """
+    )
+    return auth_file
+
+
+@pytest.fixture
+def multiple_credentials(tmp_path: Path) -> Path:
```

Comment:
These two new fixtures should be used in both test_cli and test_configure

Reply:
ok but at least we should test that `logfire whoami` doesn't ask the user for a region when there are multiple credentials.